tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/AddressControlTest.java

```diff
@@ -282,7 +281,7 @@ public void testGetNumberOfBytesPerPage() throws Exception {
       session.createQueue(address, address, true);
 
       AddressControl addressControl = createManagementControl(address);
-      Assert.assertEquals(ActiveMQDefaultConfiguration.getDefaultJournalFileSize(), addressControl.getNumberOfBytesPerPage());
```

Comment:
my previous question also applies here.

Reply:
It's not semantically right to compare `pageSizeBytes` with default journal file size. The default page size should be AddressSettings.DEFAULT_PAGE_SIZE, see https://github.com/apache/activemq-artemis/blob/eb3c4931c2871a58fea9208eebaf90bc8ffbe462/artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java#L514